setup.py

```diff
@@ -31,18 +31,19 @@
     ],
     python_requires='>=3.6',
     install_requires=[
-        "github3.py>=1.3",
+        "github3.py>=3.2.0",
```

Comment:
There's a potential [dependency confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610) vulnerability here (consider if someone gains access to github3.py's PyPI entry and uploads version 9000 with an exploit; the unconstrained `>=3.2.0` will pick it up and run malicious code).

Safer to say: `github3.py ~= 3.2`